test/buildah-bud/apply-podman-deltas

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# *** NOTE TO READER: Please skip down to "user-customizable section" below!
```

Comment:
Review recommendation: this advice is suitable for reviewers, too. If you want, give a once-over to the bash cruft immediately below, but what I really truly care about is the actual user-maintainable `errmsg` and `skip` directives. I want to know: do you find that readable? Maintainable?